src/components/headers/HeaderWrapper.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
+import * as classNames from 'classnames';
+import { Content, IContentProps } from '../content/Content';
+import * as _ from 'underscore';
+import { ITabsHeaderProps, TabsHeader } from './TabsHeader';
+
+export interface IHeaderWrapperProps extends ITabsHeaderProps, React.ClassAttributes<React.Component<any, any>> {
+  description?: string;
+  actions?: IContentProps[];
+  classes?: string[];
+}
+
+export class HeaderWrapper extends React.Component<IHeaderWrapperProps, {}> {
+
+  private getClasses(): string {
+    return classNames('flex flex-center space-between mod-header-padding header-height pb2',
```

Comment:
consider putting each argument on its own line, code standards say it

Reply:
Its a string. why ?